.prettierignore

```diff
@@ -0,0 +1,12 @@
+**/.git
+**/.github
+**/.svn
+**/.hg
+**/node_modules
+
+# Ignore artifacts:
+build
+coverage
+
+**/*.css
+**/*.md
```

Comment:
I think `*.css` will match any files ending with `.css`, also please don't forget to update `*.css` to `*.scss` if you decide to use `sass` later

```suggestion
*.css
*.md
```